gips/0001-gip-process.md

```diff
@@ -31,84 +31,137 @@ The GIP process includes both the stages and acceptance criteria that a proposal
 
 The high-level process is captured in the diagram below.
 
+(TODO: update this diagram)
 ![GIP Process Flowchart](../assets/gip-0001/gip-process-flowchart.png)
 
 ## Before a proposal is written
 
-The author(s) should do the leg work to assess whether their idea is a good one that is likely to be supported by the community. This includes discussing the idea in public formus such as Discourse, Discord and Twitter.
+The author(s) should do the leg work to assess whether their idea is a good one that is likely to be supported by the community. This includes discussing the idea in public spaces such as Discord, Twitter, and The Graph forum.
```

Comment:
we could add some links to all of this.
Forum: https://forum.thegraph.com/
Discord: https://discord.gg/graphprotocol
Twitter: https://twitter.com/graphprotocol 

I think we should encourage the use of these venues only in addition to the forum.